templates/index.html

```diff
@@ -47,8 +51,8 @@
       <div class="container">
         <h1>SHORT.RS</h1>
         <form method="GET" action="/new">
-            <input type="text" name="target" autofocus/>
-            <input type="submit" value="Submit"/>
+            <input type="text" name="target" placeholder="Paste URL here"  autofocus/>
```

Comment:
Only comment here is they _may_ not be pasting. I'm not sure off the top of my head what better text would be so if I can't think of anything by the time you want to put this through don't worry about it.